src/svcagt_startup_states.c

```diff
@@ -58,12 +58,12 @@ int find_services_ (const char *services_dir)
 	DIR *dirp = opendir (services_dir);
 	char svc_name[SVCAGT_SVC_NAME_BUFLEN];
 
-	if (dirp == NULL) {
-		log_errno (errno, "Could not open systemd directory %s\n", services_dir);
+	if (dirp == NULL) {		
+		svcagt_log (LEVEL_ERROR,"Could not open systemd directory %s, errno:%d\n", services_dir, errno);
```

Comment:
THis will not show the system error message associated with errno, just the number.